### DISCUSSION TITLE
Are own fe_user fields still possible?

### DISCUSSION THREAD

**Discussion by webdev2027** [oliverklee/ext-onetimeaccount#936] [Q&A] [answered]
Hi Oli, in earlier versions I use my own fe_user-fields in your extension (Login ->Ext.seminars).
tx_ontimeaccount_pi1.forms.elements.myownfefield ...

Is this still possible? I need more field in flexform, like select boxes etc.

**Comment by oliverklee** [comment, marked as answer]
Hi @webdev2027, yes, this is still possible. You'll need to XCLASS the domain model. Hopefully, I'll create a demo extension for this some day.
